ui/src/components/uploader/QasCustomUploader.vue

```diff
@@ -0,0 +1,120 @@
+<script>
+import Pica from 'pica'
+import { QUploader } from 'quasar'
+import uploaderMixin from '../../mixins/uploader'
+import QUploaderBase from 'quasar/src/components/uploader/QUploaderBase'
+
+export default {
+  name: 'QasCustomUploader',
```

Comment:
É necessário?

Reply:
Se algum dia formos usar um `<component :is>` ele necessita ter um `name`, isso funciona como uma key